R/cacoa.R

```diff
@@ -282,9 +282,10 @@ Cacoa <- R6::R6Class("Cacoa", lock_objects=FALSE,
       min.cells.per.sample=10, min.samp.per.type=2, min.gene.frac=0.01,
       ref.level=self$ref.level, sample.groups=self$sample.groups,
       verbose=self$verbose, n.cores=self$n.cores, name="expression.shifts",
-      n.permutations=1000, genes=NULL, n.pcs=NULL, top.n.genes=NULL, ...) {
+      n.permutations=1000, genes=NULL, n.pcs=NULL, top.n.genes=NULL, saved.folder = saved.folder, ...) {
```

Comment:
Hi @reyvnth. Not sure why you want to introduce a parameter to save CMs? Users could do this themselves later, this is not the scope of this function.